mpp/mpp.cpp

```diff
@@ -626,6 +638,9 @@ MPP_RET Mpp::control_mpp(MpiCmd cmd, MppParam param)
         MppPollType block = *((MppPollType *)param);
         mOutputBlock = block;
     } break;
+    case MPP_SET_OUTPUT_BLOCK_TIMEOUT: {
+        mOutputBlockTimeout = *((int *)param);
```

Comment:
I would suggest to use 64 bits as it is more common type for the milisecond in the other media platform.